distributions/aws/aws-telemetry/cronjob.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: aws-kubeflow-telemetry
+spec:
+  schedule: "0 0 * * *"
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 3
+  failedJobsHistoryLimit: 0
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 0
+      backoffLimit: 3
+      template:
+        metadata:
+          annotations:
+            sidecar.istio.io/inject: "false"
+        spec:
+          restartPolicy: Never
+          containers:
+          - name: amazonlinux
+            image: public.ecr.aws/amazonlinux/amazonlinux:2
+            command:
+              - /bin/sh
+              - -c
+              - |
+                get_instance_id() {
+                  local _token
+                  _token=$(curl -s --retry 3 --max-time 3 -X PUT http://169.254.169.254/latest/api/token -H "X-aws-ec2-metadata-token-ttl-seconds: 21600")
+                  if [[ -n ${_token+x} ]]; then
```

Comment:
what does` +x ` mean here?

Reply:
https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash